### DISCUSSION TITLE
My test ran a few steps and quit.

### DISCUSSION THREAD

**Discussion by Adairle** [idaholab/moose#22403] [Q&A Getting Started] [answered]
Thank you for your reply
Environment Name Not Found : Could not find  conda environment: moose
You can list all discoverable environments with 
But when l run the command "mamba activate mouse". The above situation will occur.
@GuidGuid

**Comment by GiudGiud** [comment, marked as answer]
Hello

You need to follow the instructions from the beginning. Likely the environment did not get created, or you skipped the 'close and reopen your console' part. 
https://mooseframework.inl.gov/getting_started/installation/conda.html
Also please do not open new posts for the next reply. 

Guillaume
